Source/SPUD/Private/SpudSubsystem.cpp

```diff
@@ -680,22 +667,15 @@ void USpudSubsystem::UnloadStreamLevel(FName LevelName)
 
 	if (StreamLevel)
 	{
-		PreUnloadStreamingLevel.Broadcast(LevelName);
```

Comment:
This was moved to after the loaded level is confirmed valid. It seems premature to broadcast it before then (one reason being that the matching PostUnloadStreamingLevel would never fire).